packages/samples/vuejs/src/assets/base.css

```diff
@@ -0,0 +1,74 @@
+/* color palette from <https://github.com/vuejs/theme> */
```

Comment:
It will be important to clean up the base css, icons, logo, fonts, etc that comes "pre installed" with VueJS cli tooling as much as possible before marking this project as "complete".

The goal is to provide a sample with as less clutter as possible so that someone that wants to learn will only see the absolute necessary code that's needed for Atomic + VueJS setup (and not everything that would be needed for a full fledge website).

This can be done later (just a reminder) !